vnext/ReactWindowsCore/packages.ReactWindowsCore-Desktop.config

```diff
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="boost" version="1.68.0.0" targetFramework="native" />
-  <package id="ReactNative.Hermes.Windows" version="0.1.6" targetFramework="native" Condition="$(USE_HERMES)' == 'true'" />
```

Comment:
I'm not sure how you're making sure that the desktop project installs the V8/Hermes packages if there isn't a similar packages.config file for a desktop-specific static library? Can you clarify what is the mechanism that enables this?
Also, we should make sure that creating a new app and building it, does not cause us to download/include V8/Hermes packages.

Reply:
That's correct. I was missing adding it to the Desktop project.
Solved.